submarine-server/server-rpc/src/main/java/org/apache/submarine/server/rpc/SubmarineRpcServer.java

```diff
@@ -109,11 +116,38 @@ private static ClientContext getClientContext(RpcContext rpcContext) {
     ClientContext clientContext = new ClientContext();
     clientContext.setYarnConfig(conf);
     mergeSubmarineConfiguration(clientContext.getSubmarineConfig(), rpcContext);
+    ClassLoader classLoader = new URLClassLoader(constructUrlsFromClasspath("../lib/submitter"));
```

Comment:
The `../lib/submitter` path will store `submitter-yarn` and `submitter-k8s` separately,
If we load all jars in the `../lib/submitter` path at the same time, there may be conflicts.
I think the RpcServer only needs to load the jar in the `submitter-yarn` or `submitter-k8s` path.
What do you think?

Reply:
Yeah, agree with you. The code just adds jars in the submitter, not including jars in the submitter/submitter-k8s